cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,81 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('/');
   });
 
-  it('', () => {
+  it('Student Registration Form', () => {
+    cy.debug('[crossorigin="CORS"]');
+    cy.get('form').get('input[placeholder="First Name"]').type('testName');
```

Comment:
Please use randomly generated data each time. Create a generator in a separate file (e.g. support or config).